.travis.yml

```diff
@@ -1,49 +1,99 @@
 language: python
-sudo: required
-services:
-  - xvfb
-matrix:
-    include:
-    - python: "3.6"
-      env: DEPS="numpy scipy cython nose matplotlib scikit-learn gstools"
-    - python: "3.5"
-      env: DEPS="numpy scipy=0.18 cython nose matplotlib scikit-learn=0.18.1"
-    - python: "2.7"
-      env: DEPS="numpy=1.10.4 scipy=0.17 cython nose matplotlib scikit-learn=0.17.1"
-
-# setup adapted from https://github.com/soft-matter/trackpy/blob/master/.travis.yml
+python: 3.8
+
+# setuptools-scm needs all tags in order to obtain a proper version
+# git:
+#   depth: false
+
+env:
+  global:
+    - TWINE_USERNAME=geostatframework
+    - CIBW_BEFORE_BUILD="pip install numpy==1.17.3 scipy==1.3.2 cython==0.29.14 setuptools"
+    - CIBW_TEST_REQUIRES="pytest scikit-learn"
+    - CIBW_TEST_COMMAND="pytest -v {project}/tests"
+
 before_install:
-    - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
-    - chmod +x miniconda.sh
-    - ./miniconda.sh -b
-    - export PATH=/home/travis/miniconda3/bin:$PATH
-    - conda update --yes conda
-    - conda info -a
-    - conda config --add channels conda-forge
-
-install:
-    - conda create -q -n krige-env -y $DEPS pip pytest-cov pytest python=${TRAVIS_PYTHON_VERSION}
-    - source activate krige-env
-    - which pip
-    - pip install coveralls
-    - pip install -e .
-
-before_script:
-  - "export DISPLAY=:99.0"
-
-# command to run tests
+  - |
+    if [[ "$TRAVIS_OS_NAME" = windows ]]; then
+        choco install python --version 3.8.0
+        export PATH="/c/Python38:/c/Python38/Scripts:$PATH"
+        # make sure it's on PATH as 'python3'
+        ln -s /c/Python38/python.exe /c/Python38/python3.exe
+    fi
+
 script:
-    - source activate krige-env
-    - pytest -sv ./pykrige/ --cov=pykrige
-    - |
-        # run examples
-        # scikit-learn needed by the regression kriging example
-        # conda install -y scikit-learn gstools
-        set -x
-        for f in examples/*.py; do
-           python $f 2>&1 | tee -a ~/log.txt
-        done
-        if grep -q "Traceback (most recent call last):" ~/log.txt; then false; else true; fi
-
-#after_success:
-#    coveralls
+  - python3 -m pip install cibuildwheel==1.3.0
+  - python3 -m cibuildwheel --output-dir dist
+
+after_success:
+  - |
+    if [[ $TRAVIS_PULL_REQUEST == 'false' ]]; then
+      python3 -m pip install twine
+      python3 -m twine upload --verbose --skip-existing --repository-url https://test.pypi.org/legacy/ dist/*
```

Comment:
Do we really want to do this on each commit on master?

Reply:
That is what `test.pypi` is there for. See for example: https://test.pypi.org/project/gstools/#history
It's a good way to check, if the wheels were build correctly and nothing was broken with the current commit.

Reply:
BTW: so I found an error, that pypi cant render the `raw` directive of RST ;-)